modules/lang-groovy/src/test/resources/rest-api-spec/test/lang_groovy/30_compile_limit.yaml

```diff
@@ -0,0 +1,55 @@
+setup:
+  - do:
+      indices.create:
+          index:  test
+  - do:
+      index:
+          index:  test
+          type:   test
+          id:     1
+          body:   { foo: bar }
+          refresh: wait_for
+
+---
+teardown:
+  - do:
+      cluster.put_settings:
+        body:
+          transient:
+            script.max_compilations_per_minute: 15
```

Comment:
How does this interact with the cluster wide setting you set in `modules/lang-groovy/build.gradle`? I expect when any of these tests run it'll reset the setting to 15. And since these run in a random order, this could cause random failures? Maybe this should just null the setting so it goes back to the default from the file?